lib/workers/repository/init/vulnerability.ts

```diff
@@ -184,9 +184,15 @@ export async function detectVulnerabilityAlerts(
           matchFileNames,
         };
 
+        let matchCurrentVersion = `< ${val.firstPatchedVersion}`;
+        if (datasource === MavenDatasource.id) {
```

Comment:
This should be extended by `|| datasource === NugetDatasource.id` , as otherwise nuget fix PRs are no longer created / existing ones closed.